bookkeeper-server/src/test/java/org/apache/bookkeeper/proto/NetworkLessBookieTest.java

```diff
@@ -68,12 +71,11 @@ public void testUseLocalBookie() throws Exception {
         }
 
         for (BookieServer bk : bs) {
-            for (ChannelManager channel : bk.nettyServer.channels) {
-                if (! (channel instanceof VMLocalChannelManager)) {
+            for (Channel channel : bk.nettyServer.allChannels) {
```

Comment:
This test will not be valid because of the reasons I described above, with the allChannels being stomped on.  I would suggest you also check that eventLoopGroup is null.